application/core/GlobalHelpers.php

```diff
@@ -9,207 +8,208 @@
  * is sent, or cookieDomain if no parameter
  * is sent
  * 
- * @param	string
- * @author	sbebbington
- * @date	27 Jul 2017 15:19:51
- * @version 0.1.4-RC2
- * @return	string
- * @throws  \Application\Core\FrameworkException\FrameworkException
+ * @param   string
+ * @author  sbebbington
+ * @date    27 Jul 2017 15:19:51
+ * @version 0.1.4-RC3
+ * @return  string
+ * @throws  FrameworkException
  */
 function getConfig(string $parameter= 'cookieDomain'){
-	if(!file_exists(serverPath("/config/site.json"))){
-		throw new FrameworkException("A site.json file is required in the configuration at the application level for the framework to run", "0x02");
-	}
-	if(empty($parameter)){
-		$parameter = 'baseURL';
-	}
-	return json_decode(file_get_contents(serverPath("/config/site.json")), true)[$parameter] ?? null;
+    if(!file_exists(serverPath("/config/site.json"))){
+        throw new FrameworkException("A site.json file is required in the configuration at the application level for the framework to run", "0x02");
+    }
+    if(empty($parameter)){
```

Comment:
$parameter defaults to 'cookieDomain' i.e never empty

Reply:
Good point well made. Thanks.